catalog/spatial/registry/registry-admin-modules/registry-admin-remote-ui/src/main/webapp/js/HandlebarsHelpers.js

```diff
@@ -0,0 +1,188 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+/*global define*/
+define([
+        'icanhaz',
+        'underscore',
+        'handlebars'
+    ],
+    function (ich, _, Handlebars) {
+        "use strict";
+
+        // The module to be exported
+        var helper, helpers = {
```

Comment:
Instead of having a single `helpers` module, could we split this up into separate ones (e.g., `Conditionals` and `StringUtils)? It might help people figure out what the purpose of the module is and prevent it from becoming a dumping ground for all sorts of utility functions in the future.